src/index.css

```diff
@@ -27,17 +27,32 @@ body[data-chain-type='testnet'] {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
-  font-family: Lato, sans-serif, Montserrat, "PingFang SC", -apple-system;
+  font-family: Lato, sans-serif, Montserrat, 'PingFang SC', -apple-system;
 }
 
 .monospace {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
 }
 
 *[role]:focus {
   outline: none;
 }
 
+/* stylelint-disable-next-line selector-class-pattern */
+.ant-tabs-tab.ant-tabs-tab-active {
```

Comment:
Is this issue fixed by moving the overridden to the global context?

Reply:
> Is this issue fixed by moving the overridden to the global context?

It may have been fixed using the recommended and more standardized approach by antd, using the `TabPane` component.

The class name here is likely due to the original CSS module being converted to a styled-components implementation and not having a suitable place to put it, so it was moved here.

However, it is not quite appropriate to have it here. This is more like a theme customization. I suggest avoiding global theme overrides as much as possible and keeping them limited to encapsulated components, such as within ScriptTab.

Reply:
I put this here because I saw other `ant style` are defined here or other file as `global`. 
Maybe I think there should be another PR to refactor these.

Reply:
> I put this here because I saw other `ant style` are defined here or other file as `global`.

It was originally `.scriptTabs :global { .ant-tabs-tab-active { ... } }`, so it is a local change that should only affect components under `.scriptTabs`. It should not be considered as a global change.

> Maybe I think there should be another PR to refactor these.

If there are some common theme changes, they can be encapsulated into themed Ant Design components.

However, I'm not sure if this situation occurs frequently enough to warrant it.

Reply:
> > I put this here because I saw other `ant style` are defined here or other file as `global`.
> 
> It was originally `.scriptTabs :global { .ant-tabs-tab-active { ... } }`, so it is a local change that should only affect components under `.scriptTabs`. It should not be considered as a global change.
> 
> > Maybe I think there should be another PR to refactor these.
> 
> If there are some common theme changes, they can be encapsulated into themed Ant Design components.
> 
> However, I'm not sure if this situation occurs frequently enough to warrant it.

Not only components under `.scriptTabs` changed, I tried it, so I just put them here

Reply:
> Not only components under `.scriptTabs` changed, I tried it, so I just put them here

 You can refer to the compiled CSS result.

### Source code
```scss
.scriptTabs {
  display: flex;
  flex-direction: column;
  justify-content: center;

  :global {
    .ant-tabs-nav {
      &::before {
        border-bottom-width: 4px;
      }
    }

    .ant-tabs-nav-list {
      padding-left: 40px;
    }

    .ant-tabs-tab-btn {
      color: #333;
      font-weight: 400;
      font-size: 20px;
      line-height: 23px;
      margin-bottom: 0;

      &[aria-selected='true'] {
        font-weight: 500;
      }
    }
  }
}
```
### Build result
```css
.styles_scriptTabs__K3ITv {
  display: flex;
  flex-direction: column;
  justify-content: center;
}
.styles_scriptTabs__K3ITv .ant-tabs-nav::before {
  border-bottom-width: 4px;
}
.styles_scriptTabs__K3ITv .ant-tabs-nav-list {
  padding-left: 40px;
}
.styles_scriptTabs__K3ITv .ant-tabs-tab-btn {
  color: #333;
  font-weight: 400;
  font-size: 20px;
  line-height: 23px;
  margin-bottom: 0;
}
.styles_scriptTabs__K3ITv .ant-tabs-tab-btn[aria-selected=true] {
  font-weight: 500;
}
```

Reply:
I just put them to the specific components